---
title: "Final Report: Impact of Natural Birth Delivery and Vegetarian Diet on Improving Children’s Gut Microbiome Against Chronic Diseases"
author: "Sanam Sajjadi"
date: "December 5, 2018"
output:github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

The human gastrointestinal (GI) tract plays an important role within the body as the main area for nutrient breakdown and absorption. With the introduction of the Human Microbiome project, increasing light has been shed on the vast amount of microbial life that resides within the GI tract, and the overall effects that it has on the health of the human body. Investigations of the gut microbiome cover topics such as the differences in the microbiome of infants born naturally or through a cesarean section [@azad2013gut; @dominguez2010delivery; @goedert2014diversity]; the influence of the microbiome on obesity rates [@doshuman; @koleva2015infant; @ley2005obesity; @xu2015dietary]; differences based on geographical region or biological sex [@dominianni2015sex; @suzuki2014geographical]; and the microbiome’s correlation with dysbiosis or diseases [@backhed2012defining; @de2010impact; @larsen2010gut; @tilg2011gut]. Therefore, through current research of microbiome differences between populations using fecal samples, animal models, and sequencing for 16s rRNA, researchers will be able to better understand the microbial role that the gut microbiome plays in the overall health and physiology of the human body.


There are many studies that looked at the variances between the gut microbes of babies that were born vaginally and those who were born through a Cesarean section (C-section). This has been a prominent area of study as researchers try to determine if the route of birth, and the consequential first type of bacterial colonization will later result in altered disease rates such as asthma, allergies, and obesity [@azad2013gut]. From these studies it has been determined that the infants birthed vaginally harbor bacterial species most similar in relation to the bacterial species present within their mother’s vagina [@dominguez2010delivery]. This study also found that the infants born through a C-section show bacterial species most common to those found in human skin flora, such as *Staphylococcus spp* [@dominguez2010delivery]. Another study also found that C section infants also have a much lower abundance of *Bacteroides* and *Bifidobacteria* leading to an increased chance of colonization of pathogenic *Clostridium difficile* [@goedert2014diversity]. Also integrated into these studies is that breast feeding an infant also is shown to promote what is classified as a healthy gut microbiome. Breast milk is shown to provide selective metabolic substrates to help beneficial bacteria to colonize infant guts [@azad2013gut]. Even as the infants grew to adults, fecal samples showed differences in the gut microbiome of individuals who were born vaginally and those born by C-section [@goedert2014diversity]. These studies show the importance that the first types of bacterial colonization can have on infants, and how this can play a role later in their lives. 


Another large area of gut microbiome research is differences of microbial composition between lean individuals in comparison to those with obesity. Many of these studies use obese mice to compare them to lean mice, or germ free mice are used and gut microbes from an obese and lean phenotype are transplanted into the mice and rates of weight and adipose gain are observed [@koleva2015infant; @xu2015dietary]. It has been shown that the two main bacterial genera that are most responsible for contributing to whether an individual will remain lean or become obese are Bacteroidetes and Firmicutes It is also noted in that there is supporting evidence that shows that *Bifidobacterium spp*. and *Methanobrevibacter smithii* are associated with lower BMI and overall adipose ratios [@koleva2015infant]. In obese individuals levels of Firmicutes are much higher in ratio to Bacteroidetes, this is said to contribute to increased levels of adipose formation through the ability to more efficiently harvest energy from food [@ley2005obesity].
The elevated ratios of Firmicutes allows for increase in intestinal monosaccharide absorption, as well as increasing de novo lipogenesis (Tilg and Kaser 2011). Some research has also found that reduced microbial diversity within the gut can also contribute to obesity by shifting the bacteria that are able to colonize and the abundance that the colonies are able to form [@koleva2015infant; @tilg2011gut]. Based on the correlations between the gut microbiome and obesity, it opened the door for further studies into how the microbiome can change based on other important factors. 

Much research on the gut microbiome has focused on Western culture and the effects that it has on the bacterial composition of the Gastrointestinal tract. Through cross-cultural research it has been determined that similar phyla of bacteria are generally present within human guts, although every gut microbiome’s exact makeup is inherently unique [@jumpstart2012evaluation]. These common phyla are: Firmicutes, Bacteroidetes, Actinobacteria, Proteobacteria, Verrucomicrobia, TM7, Spirochaeates, and Fusobacteria [@doshuman; @turnbaugh2009effect]. Although the normal phyla seen within the GI tract remains similar amongst populations, changes to the configuration of bacteria can change rapidly in response to diet changes, especially when changes are compared between the normal Western diet that largely consists of high fat and high sugar content, and plant polysaccharide-rich, low fat diet [@turnbaugh2009effect]. Studies have been performed on mice in which Bacteroidetes from lean mice were transplanted into obese mice to test the effects diet on the gut microbiome. When the post transplantation obese mice were fed a diet high in fruits and vegetables the mice showed a decrease in adipose formation, and the gut microbiome began to more closely resemble that of the lean mouse. However, when the post-transplantation mouse was fed a diet high in saturated fats, the microbiome slowly began to resemble the original mouse’s gut ( [@xu2015dietary]. It should also be noted that what constitutes a healthy microbiome may differ based on geographical regions and the nutritional content and demands of the region, this may be able to allow researchers to identify variables present in the environment that allowed a regions gut microbiome to be sculpted the way that it is today[@suzuki2014geographical]. Along with the region and diet being a factor in the gut microbiome composition there is evidence that biological sex and the associated hormones also plays a role in determining the microbial ratios. Evidence supports that women have an overall lower ratio of Bacteroidetes in comparison to men, the study found that until puberty the gut microbiomes between men and women had very little differences, and during puberty the differentiation began.This supports the present hypothesis that sex hormone levels may play a role in maturing the gut microbiome into the differences seen between male and female microbiomes  [@dominianni2015sex]. These areas of study support that there is a base of common phyla seen across almost all human gut microbiomes, within individual variation based on a variety of factors. 


The gut microbiome is often described as an essential ‘organ’ based on the notion that it provides the host with immune system training, pathogen protection, and disease protection. The host immune system and the gut microbiome are interconnected as the immune system shapes the overall composition and bacterial diversity within the gut, while the microbes within the gut provide signals allowing for immune development, and allow for immune responses allowing for the prevention of pathogens or diseases [@hooper2012interactions]. Another influencing factor for the determination of disease prevalence is host genetic factors that can influence microbial colonization. The subsequent colonization patterns some leading to dysbiosis and the bacterial gut ratios can ultimately contribute to the disease process [@backhed2012defining; @benson2010individuality]. A common disease that is associated with the gut microbiome is irritable bowel disease (IBD), which is associated with a decrease in gut microbes due to the use of antibiotics, and over sanitization in children who are still developing their GI microbiome [@de2010impact].  Another disease that has been an area of recent study is diabetes and the microbial influence on its development. When the development of type-1 diabetes was observed in rats, there were higher amounts of *Bacteroides spp*. found within the gut flora, in comparison to non-diabetic rats [@larsen2010gut]. This study also found that there were lower amounts of Firmicutes, while there were increased rates of colonization for Bacteroidetes and Proteobacteria within diabetic individuals when their gut microbiomes were compared with non-diabetics. There is also evidence from another study that has found similar observations that within individuals with type-2 diabetes there is an increase of Gram-negative bacteria, Bacteroidetes being noted [@tilg2011gut]. The increased levels of Bacteroidetes have been correlated to the reduced glucose tolerance, a marker for diabetes pathogenesis [@larsen2010gut]. When Bacteroidetes concentrations are decreased in diabetic mice there is a decrease in systemic inflammation and improved metabolic endotoxemia [@tilg2011gut]. With increased knowledge of how the gut microbiome effects diseases, there is a greater potential for the discovery of possible therapeutics that could target the gut and treat disease on a microbial level. 



Through the expansion of microbiome research the gut microbiome has been brought to the forefront of human health. Through studies it has been shown that there is a significant difference in the types of microbes that colonize an infant born either vaginally or by C-section, and these differences follow the infants into adulthood and increase their rates of asthma or allergies. Other studies have also shown that different ratios of the common gut microbiome phyla Bacteroidetes and Firmicutes can increase an individual’s chance of being obese or lean. There are also diet and geographical effects that play a role in the colonization of gut microbes, as well as the ratios of the common phyla. An important role of the gut microbiome not to be overlooked is the rates of disease prevalence and progression through the differences of microbial colonies present within the GI tract. Within this report of De Filippo et al.’s research I will investigate the differences of microbiomes within individuals from two different populations from varying geographical regions, whose diets and birth delivery methods are also subsequently different. I will predict that vegetarian diet and natural birth thorough the maternal birth canal would prevent obesity, chronic diseases and allergies in these children.





# Methods

## Study design


## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r load-libraries, message = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the install.packages() function
# or with the 'Packages' pane in RStudio
# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
```
```{r load-other-libraries}
# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")
# This to export a fasta of our final denoised sequence variant
library("seqinr")
# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")
# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")
# Be sure to load in the phyloseq data object created
load("output/phyloseq_obj.RData")
stopifnot(exists("phyloseq_obj"))
```
```{r load-packages-and-data}
# load packages as needed
library("dplyr")
load("output/phyloseq_obj.Rdata")
# load data produced from analysis scripts using
# something like load("output/processed_data.Rdata")
```



```{r make-mediterranean-diet-plot}
# We want to see the impact of diet
# On gut microbiome and as there are
# So many sequences in this data set
# Now we want to subset this data
# And look At gut microbiome of children with
# Mediterranean diet
# We look at the Phylum level
mediterranean_phyloseq_obj <- subset_samples(phyloseq_obj,
                              special_diet == "mediterranean diet")
plot_bar(mediterranean_phyloseq_obj, x = "Phylum")
```


**Figure1**: This figure is a plot bar that shows the different phyla of gut microbiome in children with Mediterranean diet.

Most of the bacteria were from Firmicutes phylum.


```{r make-vegeterian-diet-plot}
# We want to compare Mediterranean diet
# With vegeterian diet and
# AS we have a large data set
# We want to subset our dataset
# To look at gut microbiome of children
# With vegeterian diet
# Again we look at Phylum level
vegeterian_phyloseq_obj <- subset_samples(phyloseq_obj,
special_diet ==
"vegeterian diet (derived from sorghum, millet, black eyed pea)")
plot_bar(vegeterian_phyloseq_obj, x = "Phylum")
```

**Figure2**: This figure is a plot bar that shows the different phyla of bacteria in children with vegeterian diet.

Most of the sequences where from Bacteroidiates phylum.



```{r make-natural-childbirth-plot}
# Now we want to see the impcat of birth delivery method
# On Children's guts microbiome
# We subset our data set
# To look at gut microbiome of children
# That were born naturally
# We look at Phylum level
natural_phyloseq_obj <- subset_samples(phyloseq_obj,
birth_delivery == "natural childbirth")
plot_bar(natural_phyloseq_obj, x = "Phylum")
```


**Figure3**: This figure is a plot bar that shows the different phyla of bacteria in children that were born naturally.

Most of the bacteria were from Firmicutes and Bacteroidetes phyla.


```{r make-Cesarean-plot}
# now we want to look at microbiome of
# children's gut that were born with Cesarean
# As our data set is big, we make a new subset 
# For cesarean and then we look
# At different Phyla
cesarean_phyloseq_obj <- subset_samples(phyloseq_obj,
birth_delivery == "cesarean childbirth")
plot_bar(cesarean_phyloseq_obj, x = "Phylum")
```

**Figure4**: This figure is a plot bar that looks at different phyla of bacteria in children who were born with Cesarean section.

Most of the bacteria were from Firmicutes phylum.




```{r make-geographic-Burkina-plot}
# We want to see the impact of geographical location
# So we have to to subset our data for specific region
# Now we subset our big data
# To just look at Burkina Faso
# We look at Phylum
geographic_phyloseq_obj <- subset_samples(phyloseq_obj,
geographic_location_country ==
"Burkina Faso:Boulkiemde province,Boulpon")
plot_bar(geographic_phyloseq_obj, x = "Phylum")
```

**Figure5**: This figure is a plot bar that looks at different phyla of bacteria in children from Burkina Faso.

Most of the bacteria were from Bacteroidetes phylum.




```{r make-geographic-Florence-plot}
# After Burkina, now we want to look at 
# Gur bacteria in children from Florence
# So we have to subset our data to see different
# Phyla of bacteria in their gut
# We look at Phylum level
geographic_phyloseq_obj <- subset_samples(phyloseq_obj,
geographic_location_country == "Italy:Toscany, Florence")
plot_bar(geographic_phyloseq_obj, x = "Phylum")
```


**Figure6**: This figure is a plot bar that shows different phyla of bacteria in children from Florence.

Most of the bacteria were from Firmicutes phylum.






In addition to a minimum of 5-8 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
